README.md

```diff
@@ -1,2 +1,43 @@
-# KOBAW
-KOB Association Wallet
+
+| Title | KOB Association Wallet |
+|--- | ---|
+| Version | TBD |
+
+
+# Abstract   
+
+The KOB Association Wallet is an association/organisational wallet for the city of Kochi. This holds the verifiable credentials about registered entities (organisations) under a particular jurisdiction. And this is a Web Application hosted on the Community Cloud for the City of Kochi. A wallet provides an interface for storing and accessing identity information.
+
+# Dependent Projects
+
+*TBD*
+
+# Motivation
+
+The KOB Association wallet is a new service uses cutting-edge blockchain-based technology, which provides the foundation to help the organisations and associations in city of Kochi to be digitalized. This project is a part of Kochi OrgBook that is aimed to streamline and modernize services for Kochi.
```

Comment:
KOBAW is not a service but rather a wallet application which leverages blockchain technology to store credentials for each association in the city of Kochi.